Assets/MRTK/Core/Utilities/DebugUtilities.cs

```diff
@@ -30,27 +56,110 @@ public static void DebugAssert(bool condition)
         /// Logs an error message.
         /// </summary>
         /// <param name="message">The message to log.</param>
+        [Obsolete("Deprecated. Use LogError instead.")]
         public static void DebugLogError(string message)
         {
-            Debug.LogError(message);
+            LogError(message);
         }
 
         /// <summary>
         /// Logs a warning message.
         /// </summary>
         /// <param name="message">The message to log.</param>
+        [Obsolete("Deprecated. Use LogWarning instead.")]
         public static void DebugLogWarning(string message)
         {
-            Debug.LogWarning(message);
+            LogWarning(message);
         }
 
         /// <summary>
         /// Logs a message.
         /// </summary>
         /// <param name="message">The message to log.</param>
+        [Obsolete("Deprecated. Use Log instead.")]
         public static void DebugLog(string message)
         {
-            Debug.Log(message);
+            Log(message);
+        }
+
+        /// <summary>
+        /// Logs an error message.
+        /// </summary>
+        /// <param name="message">The message to log.</param>
+        public static void LogError(string message)
+        {
+            if (LogLevel <= LoggingLevel.Error)
+            {
+                Debug.LogError(message);
+            }
+        }
+
+        /// <summary>
+        /// Logs a warning message.
+        /// </summary>
+        /// <param name="message">The message to log.</param>
+        public static void LogWarning(string message)
+        {
+            if (LogLevel <= LoggingLevel.Warn)
+            {
+                Debug.LogWarning(message);
+            }
+        }
+
+        /// <summary>
+        /// Logs a message at the informational level.
+        /// </summary>
+        /// <param name="message">The message to log.</param>
+        public static void Log(string message)
```

Comment:
Rename to LogInfo for consistent naming as Log+Logging Level

Reply:
I named these .Log to match the style of UnityEngine.Debug log naming (i.e. familiarity being the goal). (Which also has this inconsistency)

Would be curious to hear other's preferences on this

Reply:
Added some comments to the inside of that function to capture this intent.